project/ScalafixBuild.scala

```diff
@@ -47,7 +47,7 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
       scalaVersion.value.startsWith("2.12")
     }
     lazy val warnUnusedImports = Def.setting {
-      if (isScala3.value) Nil
+      if (scalaVersion.value.startsWith("3.3.2")) Seq("-Wunused:all")
```

Comment:
Temporarily set 3.3.2 since not yet sure which Scala 3 version will release https://github.com/lampepfl/dotty/pull/17835

Reply:
you can set it unconditionally, it won't hurt until the PR is merged, right?